### ISSUE TITLE
Regenerate to support latest protocol messages

### ISSUE THREAD

**Issue by ES-Alexander** [bluerobotics/ping-python#140]
Ping Python doesn't yet include [the new Cerulean Sonar messages](https://github.com/bluerobotics/ping-protocol/pull/206).

I believe we just need to re-run the generation, and release a new version.

**Comment by patrickelectric** [comment]
I believe that @NickNothom was working on adding support to it. Sadly I don't have one on hands to develop it.

**Comment by ES-Alexander** [comment]
Ahh, I was forgetting that because it's a new device type it'll need a `generate/templates/omniscan450.py.in` file, so it's a bit more work than just running the generation process.

**Comment by patrickelectric** [comment]
Exactly, as @Williangalvani suggested, it appears that we have the sensor here.
